services/wallet/router.go

```diff
@@ -547,6 +549,48 @@ func (r *Router) getERC1155Balance(ctx context.Context, network *params.Network,
 	return balances[0].Int, nil
 }
 
+func (r *Router) getL1Fee(ctx context.Context, network *params.Network, fromAddress common.Address, toAddress common.Address, amountIn *big.Int) (uint64, error) {
```

Comment:
I tested sending erc20 and sending eth from metamask and there is differences in l1 fees.

I beleve this logic is only valid for the case of sending eth, there should be logic for all types of tx:
Bridge / buy ENS, send ERC20, sendERC71.


This logic should be encapsulated by bridge